language/e2e-tests/src/tests/transaction_builder.rs

```diff
@@ -623,6 +624,49 @@ fn dual_attestation_payment() {
         );
         assert_eq!(output.status().vm_status().sub_status, Some(9002));
     }
+    {
+        // Transaction >= 1_000_000 threshold goes through signature verification with valid signature, passes
+        // Do the offline protocol: generate a payment id, sign with the receiver's private key, include
+        // in transaction from sender's account
+        let ref_id = lcs::to_bytes(&9999u64).unwrap();
+        // choose an amount above the dual attestation threshold
+        let payment_amount = 1_000_000u64;
+        // UTF8-encoded string "@@$$LIBRA_ATTEST$$@@" without length prefix
+        let mut domain_separator = vec![
```

Comment:
Is there some reason that this can't be written as a byte string literal? i.e., b"@@$$LIBRA_ATTEST$$@@"

Reply:
Don't think so, was just copying and pasting a different test. I'll update :)